ontology/yaml/resources/ELECTRICAL/entity_types/UPS.yaml

```diff
@@ -83,17 +83,3 @@ UPS_SS_UPSBM:
   - UPS
   - SS
   - UPSBM
-
-UPS_UPSTR:
```

Comment:
retain this - just recently created for new building, need to re-map to a canonical type instead